contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Getting started with web analytics
+sidebar: Docs
+showTitle: true
+---
+
+It's easy to get up and running with [web analytics](/web-analytics). All you need to do is [install PostHog](/docs/getting-started/install) and you'll automatically start collecting data based on traffic to your app or website.
+
+Web analytics provides you with a simple [dashboard](/docs/web-analytics/dashboard), but it can be overwhelming if you don't have a technical background or experience with marketing concepts.
+
+This post provides a gentle guide to web analytics by introducing the following:
+
+- **💡 Concepts:** explanations for common analytics terms
+- **🛠️ Features:** tools and affordances web analytics provides
+- **📋 Use cases:** practical examples of how to use web analytics
```

Comment:
A few notes:
- Docs should be straight to the point, so we should make them as concise as possible and remove all fluff
- I dont think this is a "post". Its a docs page, so you could say "this doc". Post is more like blog post/tutorial

Reply:
+1 again. In hindsight, I think this setup just adds more complexity.

Reply:
I went ahead and removed the bullet points and the reference to this doc as a "post".